examples/bzlmod/BUILD.bazel

```diff
@@ -14,7 +14,8 @@ py_library(
     name = "lib",
     srcs = ["__init__.py"],
     deps = [
-        requirement("pylint"),
+        # You can reference libraries using plain text form or the 'requirement' macro
+        "@pip//:pylint_pkg",
```

Comment:
can it just be `@pip//pylint` ? That's the nicest syntax users would enjoy most.
 (we'd generate a `external/pip/pylint/BUILD.bazel` file with appropriate aliases)

Reply:
I would love to use this and would be willing to implement this if this approach was agreed universally. @rickeylev, any opinions here?

Reply:
@alexeagle, what do you think about the inclusion of the user friendly aliases?